video/out/mac/common.swift

```diff
@@ -573,11 +573,11 @@ class Common: NSObject {
             }
             return VO_TRUE
         case VOCTRL_GET_DISPLAY_FPS:
-            let fps = data.assumingMemoryBound(to: CDouble.self)
+            let fps = data!.assumingMemoryBound(to: CDouble.self)
             fps.pointee = currentFps()
             return VO_TRUE
         case VOCTRL_GET_HIDPI_SCALE:
-            let scaleFactor = data.assumingMemoryBound(to: CDouble.self)
+            let scaleFactor = data!.assumingMemoryBound(to: CDouble.self)
```

Comment:
i really don't like the forced unwrap on data here all the time. this has implications on all future backends too. i would much prefer a guard or if let. if that's not possible i think we should find a better way.

Reply:
`VOCTRL_GET_HIDPI_SCALE` can't have a null pointer passed in, though, right? This would only be `nil` if that happened; the optional's just serving the role of C's null (but more explicitly).

Reply:
it's not about being implicitly right here, or for that matter all the other cases in the callback. it's imo bad code style and i went to great effort to minimise all forced unwraps and be as explicit everywhere as possible.